canvas_modules/harness/test_resources/parameterDefs/code_paramDef.json

```diff
@@ -239,6 +244,12 @@
         "description": {
           "default": "someofselect test"
         }
+      },
+      {
+        "parameter_ref": "hide",
+        "label": {
+          "default": "Hide 'checkbox hidden'"
```

Comment:
1. This label can be changed to "Hide code control"
2. Please copy this entire file into `canvas/canvas_modules/common-canvas/__tests__/test_resources/paramDefs/code_paramDef.json`
3. Please add a unit test for conditional code control in tearsheet panel